### PR TITLE
Cobbler option to enable boot ISOs with Secure Boot

### PR DESCRIPTION
**Pull request by keichwa**
**For a link to rendered output,** see description of https://github.com/SUSE/spacewalk/issues/25768.

### CODE REVIEW

File: modules/client-configuration/pages/autoinst-cdrom.adoc [hunk 1]
```diff
@@ -28,32 +28,41 @@ For information about creating a profile, see xref:client-configuration:autoinst
...
 cobbler buildiso --systems="system1 system2 system3" \
-  --profiles="<your-profile1-label> <your-profile2-label> <your-profile3-label> --distro=<your-distro-label>
+  --profiles="<your-profile1-label> <your-profile2-label> <your-profile3-label>" \
+  --distro=<your-distro-label>
 ----
 
+
+With [option]``--esp`` you can enable the build boot ISOs with Secure Boot.
+You explicitly specify the EFI System Partition (ESP).
+By default, {cobbler} searches for the ESP, and generates one if not found.
```
Comment (m-czernek) [inline modules/client-configuration/pages/autoinst-cdrom.adoc:57]: Well, the default Cobbler search doesn't work for SUMA, which is why we add this feature :)

```suggestion
By default, {cobbler} generates the ESP partition, which disables Secure Boot.
```

File: modules/client-configuration/pages/autoinst-cdrom.adoc [hunk 2]
```diff
@@ -28,32 +28,41 @@ For information about creating a profile, see xref:client-configuration:autoinst
...
 
 ----
 cobbler buildiso --systems="system1 system2 system3" \
-  --profiles="<your-profile1-label> <your-profile2-label> <your-profile3-label> --distro=<your-distro-label>
+  --profiles="<your-profile1-label> <your-profile2-label> <your-profile3-label>" \
+  --distro=<your-distro-label>
 ----
 
+
+With [option]``--esp`` you can enable the build boot ISOs with Secure Boot.
```
Comment (m-czernek) [inline modules/client-configuration/pages/autoinst-cdrom.adoc:55]: Should this be in past participle? We build ISOs, but the ISO is built.

```suggestion
With [option]``--esp`` you can enable the built boot ISOs with Secure Boot.
```

**Review by m-czernek — COMMENTED** [review]
Leaving a few comments. Additionally, maybe it would be helpful to add where can users find the `efi`? For example:

1. Execute `cobbler distro list` to get the distro name (already in the text):

````
suma:~ # cobbler distro list
   sles15-sp4:1:SUSE
````

2. Execute `cobbler distro report` to get information about the distro files location:

```
cobbler distro report --name sles15-sp4:1:SUSE
Name                           : sles15-sp4:1:SUSE
Architecture                   : x86_64
...output omitted...
Initrd                         : /usr/share/tftpboot-installation/SLE-15-SP4-x86_64/boot/x86_64/loader/initrd
Kernel                         : /usr/share/tftpboot-installation/SLE-15-SP4-x86_64/boot/x86_64/loader/linux
...output omitted...
```

3. See the contents of the `boot` directory. In the example above, the boot partition is the  `/usr/share/tftpboot-installation/SLE-15-SP4-x86_64/boot/x86_64/efi` file, but this can differ based on the ISO distributor.

File: modules/client-configuration/pages/autoinst-cdrom.adoc [hunk 3]
```diff
@@ -28,32 +28,41 @@ For information about creating a profile, see xref:client-configuration:autoinst
...
 ----
 cobbler buildiso --iso=/path/to/boot.iso --distro=<your-distro-label>
 ----
 
-[IMPORTANT]
-====
-You must use distro and profile labels as listed by {cobbler}, and not simply as shown in the UI.
-====
-
+You must use distribution and profile labels as listed by {cobbler}, and not simply as shown in the UI.
```
Comment (0rnela) [inline modules/client-configuration/pages/autoinst-cdrom.adoc:37]:
```suggestion
You must use distribution and profile labels as listed by {cobbler}, and not simply as shown in the {webui}.
```

**Comment by keichwa** [comment]
Thanks for your review comments!  Please check again if time permits.  TIA!

**Review by m-czernek — APPROVED** [review]
Perfect, LGTM! Can I just say, as a developer, having command output is super important when reading docs. Docs with commands but without the command outputs are pretty bad dev UX, so thank you for including that.

**Pull request merged**